tests/foreman/ui_airgun/test_os.py

```diff
@@ -9,3 +11,16 @@ def test_positive_create(session, name):
         session.os.create_operating_system({
             'name': name, 'major': major_version})
         assert session.os.search(name) == '{} {}'.format(name, major_version)
+
+
+def test_positive_create_with_ptable(session):
+    name = gen_string('alpha')
+    major_version = gen_string('numeric', 2)
+    ptable = entities.PartitionTable().create()
+    with session:
+        session.os.create_operating_system({
+            'name': name,
+            'major': major_version,
+            'ptable': {'operation': 'Add', 'values': [ptable.name]}
```

Comment:
AFAIK widgetastic provides more "readable" way for tests to use fill:
```python
create_operating_system({
    'name': name,
    'major': major_version,
    'ptable.operation' : 'Add',
    'ptable.values': [ptable.name],
})
```
Although i'm not sure about `ptable.values`, i feel like there's gotta be a way to pass `ptable.name` directly.

Could you please take a look on at least first comment? Second one may appear a separate task to figure out, so it's not a blocker.